python/doc/examples/probabilistic_modeling/random_vectors/plot_ratioOfUniforms_exploitation.py

```diff
@@ -0,0 +1,337 @@
+        boundary_1 = boundary_1_fct.draw(vSup / (uSup ** r), xSup, 101)
+        boundary_1.setColors(['blue'])
+        leg.add(r'$\varphi(\mathcal{F}_1)$')
+        g.add(boundary_1)
+    if vInf / (uSup ** r) < uSup:
+        boundary_2_fct = ot.SymbolicFunction('x', '(' + str(vInf) + '/x)^(1/' + str(r) + ')')
+        boundary_2 = boundary_2_fct.draw(xInf, vInf / (uSup ** r), 101)
+        boundary_2.setColors(['red'])
+        leg.add(r'$\varphi(\mathcal{F}_2)$')
+        g.add(boundary_2)
+    boundary_3 = ot.Curve([[vInf / (uSup ** r), uSup], [vSup / (uSup ** r), uSup]])
+    boundary_3.setColor('green')
+    g.add(boundary_3)
+    leg.add(r'$\varphi(\mathcal{F}_3)$')
+    g.setLegends(leg)
+
+    return g
+
+# %%
+# We create the function :math:`\log f` defined by:
+#
+# .. math::
+#
+#    \log f(x) = \log(\cos(x)) + x,  \quad  x \in [-\dfrac{\pi}{2}, \dfrac{\pi}{2}]
+#
+# and the :class:`~openturns.experimental.RatioOfUniforms`.
+
+
+log_UnscaledPDF = ot.SymbolicFunction('x', 'log(cos(x)) + x')
+rangePDF = ot.Interval(-pi / 2.0, pi / 2.0)
+ratioAlgo = otexp.RatioOfUniforms(log_UnscaledPDF, rangePDF)
+
+# %%
+# We draw the boundary :math:`\partial A_{f,r}` and the bounding box :math:`\partial \tilde{A}_{f,r}`
+# in the :math:`(u,v)`-space.
+g_UV = draw_RatioOfUniforms_UV(ratioAlgo)
+g_UV.setTitle(r'$\log f(x) = \log(\cos(x)) + x$, $r = $%.3g' % ratioAlgo.getR() + r', $\tau = $%.3g' % ratioAlgo.getAcceptanceRatio())
+
+# %%
+# We draw the boundary :math:`\partial \varphi(A_{f,r})` and the bounding box :math:`\partial \varphi(\tilde{A}_{f,r})`
+# in the :math:`(x,y)`-space.
+g_XY = draw_RatioOfUniforms_XY(ratioAlgo, 1001)
+g_XY.setTitle(r'$r = $%.3g' % ratioAlgo.getR() + r', $\tau = $%.3g' % ratioAlgo.getAcceptanceRatio())
+
+grid_default = ot.GridLayout(1, 2)
+grid_default.setGraph(0, 0, g_UV)
+grid_default.setGraph(0, 1, g_XY)
+grid_default.setTitle(r'$\log f(x) = \log(\cos(x)) + x$')
+view = otv.View(grid_default)
+
+# %%
+# We change the :math:`r` value and we draw the same plots.
+grid_full = ot.GridLayout(4, 2)
+for i, r in enumerate([0.5, 0.8, 1.2, 2.0]):
+    ratioAlgo.setR(r)
+    g_UV = draw_RatioOfUniforms_UV(ratioAlgo)
+    g_UV.setLegendPosition('')
+    g_UV.setTitle(r'$r = $%.3g' % ratioAlgo.getR() + r', $\tau = $%.3g' % ratioAlgo.getAcceptanceRatio())
+
+    g_XY = draw_RatioOfUniforms_XY(ratioAlgo, 1001)
+    g_XY.setLegendPosition('')
+    g_XY.setTitle(r'$\log f(x) = \log(\cos(x)) + x$, $r = $%.3g' % ratioAlgo.getR() + r', $\tau = $%.3g' % ratioAlgo.getAcceptanceRatio())
+
+    grid_full.setGraph(i, 0, g_UV)
+    grid_full.setGraph(i, 1, g_XY)
+
+grid_full.setTitle(r'$\log f(x) = \log(\cos(x)) + x$')
+view = otv.View(grid_full)
+
+# %%
+# We study the acceptance ratio :math:`\tau(r)` as a function of the parameter :math:`r` for some distributions.
+# We draw the function :math:`r \rightarrow \tau(r)`. We conclude that the default value :math:`r=1` seems to be correct
+# in all these cases.
```

Comment:
weird, according to the graphs r=0.5 leads to tau=0.265 which is better than the tau=0.227 for r=1

Reply:
Not an actual problem, as the point is to check if the acceptance ratio for r==1 is close to the optimum (let's say within 30%)